buildkit/hypopg.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+name: hypopg
+version: "1.4.0"
+homepage: https://github.com/HypoPG/hypopg
+repository: https://github.com/HypoPG/hypopg
+source: https://github.com/HypoPG/hypopg/archive/refs/tags/1.4.0.tar.gz
+description: Hypothetical Indexes for PostgreSQL
+licence: PostgreSQL
+keywords:
+  - hypopg
```

Comment:
this goes for all 3 buildkits, if you can expand a bit on the keywords (based on the content in the README), it would be appreciated.

Reply:
Forgot to update as were being used as placeholder, will do now!